tests/unit/test_job_file.py

```diff
@@ -320,8 +320,8 @@ def test_write_suite_environment(fixture_get_platform, monkeypatch):
     expected = ('\n\ncylc__job__inst__cylc_env() {\n    # CYLC SUITE '
                 'ENVIRONMENT:\n    export CYLC_CYCLING_MODE="integer"\n  '
                 '  export CYLC_UTC="True"\n    export TZ="UTC"\n\n   '
-                ' export CYLC_SUITE_RUN_DIR="cylc-run/farm_noises"\n  '
```

Comment:
When they are not under review from another direction I might spend 5 minutes fixing these strings to something more readable.